src/shared/components/Gallery/Gallery.style.ts

```diff
@@ -31,3 +34,24 @@ export const CarouselArrowsContainer = styled.div`
   grid-column-gap: ${sizes(4)};
   margin-left: auto;
 `
+
+export const TitleWrapper = styled.div`
+  align-items: baseline;
+  display: flex;
+`
+
+export const SeeAllLink = styled(Button)`
+  margin-left: ${sizes(9)};
```

Comment:
```suggestion
  margin-left: ${sizes(8)};
```
![image](https://user-images.githubusercontent.com/51168865/126626313-6ba8153e-3a31-4a97-8e47-487e4da47216.png)